Node.js/boilerplate.js

```diff
@@ -257,15 +260,74 @@ class MQBoilerPlate {
       // Set up the callback handler to be invoked when there
       // are any incoming messages. As this is a sample, I'm going
       // to tune down the poll interval from default 10 seconds to 0.5s.
-      mq.setTuningParameters( {
-        getLoopPollTimeMs: 500
-      });
+      // mq.setTuningParameters( {
```

Comment:
Remove this code and the comments that go with it. Or replace with tuning parameters that can be set.